lie about the version to rename the folder to help apk upgrade a file to
a directory

--- a/modules/python-modules/setup.py
+++ b/modules/python-modules/setup.py
@@ -59,7 +59,7 @@ if install_addons:
   requires = requires + requires_addons
 
 setup(name='syslogng',
-      version='1.0',
+      version='1.0.1',
       description='syslog-ng Python Core & Modules',
       author='Balazs Scheidler',
       author_email='bazsi77@gmail.com',
